### PR TITLE
Add MaiCoin

### PR DESCRIPTION
**Pull request by 4ndersonLin**
MaiCoin is a Taiwan cryptocurrency buy/sale site.

### CODE REVIEW

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -420,6 +420,14 @@ websites:
       - totp
     doc: https://www.luno.com/help/en/articles/1000203420
 
+  - name: MaiCoin
+    url: https://www.maicoin.com
+    img: maicoin.png
+    tfa:
+      - sms
+      - totp
+    doc: https://maicoin2.freshdesk.com/en/support/solutions/articles/32000016118-about-2-factor-authenticator
```
Comment (phallobst) [inline _data/cryptocurrencies.yml]:
```suggestion
    doc: https://maicoin2.freshdesk.com/en/support/solutions/articles/32000016116
```

**Review by phallobst — CHANGES_REQUESTED** [review]
Please replace the icon with this one which is much smaller in file size:
maicoin

**Comment by 4ndersonLin** [comment]
Thanks for the suggestions. Updated!

**Review by phallobst — APPROVED** [review]

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -420,6 +420,14 @@ websites:
...
     doc: https://www.luno.com/help/en/articles/1000203420
 
+  - name: MaiCoin
+    url: https://www.maicoin.com
+    img: maicoin.png
+    tfa:
+      - sms
+      - totp
+    doc: https://maicoin2.freshdesk.com/en/support/solutions/articles/32000016116
+
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml:432]: Let's add a region code here as well.
```suggestion
    regions:
        - tw
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**